0.Preprocess_Data/1.preprocessing_morphology_data.ipynb

```diff
@@ -898,7 +898,7 @@
                     "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">'__' will be a feature used to determine the beta coefficient
</pre><p>Is this what you mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>